app/models/qa/authorities/local/find_works.rb

```diff
@@ -0,0 +1,23 @@
+module Qa::Authorities
+  class Local::FindWorks < ActiveFedora::Base
```

Comment:
Why does this extend ActiveFedora::Base?  Shouldn't it extend `Qa::Authorities::Base`?